config/settings.yml

```diff
@@ -3,21 +3,18 @@ ssl:
   key_file: ~
   key_pass: ~
 
-solr:
-  url: ~
-
```

Comment:
the only substantive change is removing this;  everything else is whitespace and quote style